Sources/SwiftSyntaxBuilder/FunctionCallExprConvenienceInitializers.swift

```diff
@@ -31,4 +31,22 @@ extension FunctionCallExpr {
       additionalTrailingClosures: additionalTrailingClosuresBuilder()
     )
   }
+
+  public init(
```

Comment:
I think it would be more general if we changed the `ExpressibleAsIdentifierExpr` above into `ExpressibleByExprBuildable` in the convenience initializer above instead of adding another overload.